core/src/main/kotlin/com/xorker/draw/mafia/MafiaStartGameService.kt

```diff
@@ -0,0 +1,39 @@
+package com.xorker.draw.mafia
+
+import com.xorker.draw.exception.InvalidGameStatusException
+import com.xorker.draw.exception.NotFoundWordException
+import com.xorker.draw.room.RoomId
+import org.springframework.stereotype.Service
+import kotlin.random.Random
+
+@Service
+internal class MafiaStartGameService(
+    private val mafiaGameRepository: MafiaGameRepository,
+    private val mafiaKeywordRepository: MafiaKeywordRepository,
+    private val mafiaGameMessenger: MafiaGameMessenger,
+) : MafiaStartGameUseCase {
+
+    override fun startMafiaGame(roomId: RoomId) {
+        val gameInfo = mafiaGameRepository.getGameInfo(roomId) ?: throw InvalidGameStatusException
+
+        val room = gameInfo.room
+        val players = room.players
+
+        val turnList = mutableListOf<MafiaPlayer>()
+        players.forEach {
+            turnList.add(it)
+        }
+
+        val mafia = Random.nextInt(0, players.size)
```

Comment:
p5. 변수명이 조금 더 명확하면 좋을 것 같아요

```suggestion
        val mafiaIndex = Random.nextInt(0, players.size)
```